marlowe/src/Language/Marlowe/Analysis/FSSemantics.hs

```diff
@@ -188,6 +268,9 @@ symEvalVal (UseValue valId) symState =
 symEvalVal (Cond cond v1 v2) symState = ite (symEvalObs cond symState)
                                             (symEvalVal v1 symState)
                                             (symEvalVal v2 symState)
+symEvalVal (Call name _) symState =
+  M.findWithDefault (literal 0) name (symFFICalls symState)
```

Comment:
Is this assuming that FFICalls with the same name always return the same for the whole contract independently of their arguments?